SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -228,6 +228,15 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 		return false;
 	}
 
+	if (!bInitAsClient && GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
+	{
+		UReplicationGraph* RepGraph = Cast<UReplicationGraph>(GetReplicationDriver());
+		if (RepGraph == nullptr)
```

Comment:
```suggestion
		if (GetReplicationDriver()->IsA<UReplicationGraph>())
```
Is more efficient iirc (also a bit more terse and expressive I think)